src/warp.js

```diff
@@ -44,155 +49,301 @@ function getLanguage() {
 }
 
+function getTranslations(language) {
+    if (language === "de") {
+        return {
+            "aboutCloudoguToken": "Über Cloudogu",
+            "menuToken": "Menü",
+            "ecosystemLogoutToken": "EcoSystem Logout",
+            "onboardingTextToken": "Klicken Sie auf 'Menü', um ihre Tools zu sehen. Das Menü verbindet ihre Toolchain und ist von jedem Tool aus zugänglich.",
+            "onboardingHintToken": "Hinweis nicht mehr anzeigen",
+            "Development Apps": "Entwicklung",
+            "Administration Apps": "Administration",
+            "Documentation": "Dokumentation"
+        };
+    } else {
+        return {
+            "aboutCloudoguToken": "About Cloudogu",
+            "menuToken": "Menu",
+            "ecosystemLogoutToken": "EcoSystem Logout",
+            "onboardingTextToken": "Click 'Menu' to view all tools. That menu connects your toolchain and is available from any tool.",
+            "onboardingHintToken": "Do not show this hint again",
+            "Development Apps": "Development Apps",
+            "Administration Apps": "Administration Apps",
+            "Documentation": "Documentation"
+        };
+    }
+}
+
 function toggleCategory(e) {
     var target = e.target;
-    if (target && target.rel) {
-        toggleClass(target, 'warpmenu-category-open');
-        var el = document.getElementById(target.rel);
-        if (el) {
-            if (hasClass(el, 'warpmenu-collapsed')) {
-                if (lss) {
-                    localStorage.removeItem(target.rel + '.collapsed');
-                }
-                removeClass(el, 'warpmenu-collapsed');
-            } else {
-                if (lss) {
-                    localStorage.setItem(target.rel + '.collapsed', true);
-                }
-                addClass(el, 'warpmenu-collapsed');
-            }
-        }
-    }
+    toggleClass(target, 'warpmenu-category-open');
+    toggleCollapsedInStorage(target.id);
+    setCorrectColumnCount();
 }
 
-function createMenuEntry(id, entries, title, nav) {
-    var ul = document.createElement('ul');
-    ul.id = id;
-    var collapsed = false;
-    if (lss) {
-        collapsed = localStorage.getItem(id + '.collapsed');
-    }
-    if (collapsed) {
-        addClass(ul, 'warpmenu-collapsed');
+function createMenuEntry(id, entries, title, list) {
+    var category = document.createElement('li');
+    var categoryInsideContainer = document.createElement('div');
+
+    var categoryHeader = document.createElement('h3');
+    categoryHeader.innerHTML = title;
+    categoryHeader.onclick = toggleCategory;
+    categoryHeader.id = id;
+    if (isOpenCollapsible(categoryHeader.id)) {
+        addClass(categoryHeader, 'warpmenu-category-open');
     }
+
+    var categoryLinkList = document.createElement('ul');
+
     for (var i = 0; i < entries.length; i++) {
         var link = entries[i];
         var li = document.createElement('li');
         var a = document.createElement('a');
```

Comment:
```suggestion
        let link = entries[i];
        let li = document.createElement('li');
        let a = document.createElement('a');
```

Reply:
You renamed the list but leave this variables untouched. Another naming like `categoryLinkListItem` and `categoryLink` would better match the naming of `categoryLinkList`.